packages/w3up-client/test/capability/space.test.js

```diff
@@ -142,7 +142,7 @@ export const SpaceClient = Test.withContext({
         assert.equal(egressRecord.bytes, car.size, 'bytes should be the same')
         assert.equal(
           new Date(egressRecord.servedAt).getTime(),
-          Math.floor(new Date(egressData.servedAt).getTime() / 1000) * 1000,
+          Math.floor(new Date(egressData.servedAt).getTime() / 1000),
```

Comment:
**question:** What happened with these? Was it just incorrect before? I'm not seeing the corresponding implementation change.

Reply:
This is from another PR that has already been merged. I thought Github action would catch the failing test, but I don't think it is working. I just created a ticket to tackle that later (https://github.com/storacha/w3up/issues/1592), but for now, I executed the tests manually, found the broken test, and fixed.

Reply:
Weird! Well, good catch, then. 😄